docs/enterprise-config.md

```diff
@@ -0,0 +1,57 @@
+# Enterprise configuration defaults
+
+Git Credential Manager Core (GCM Core) can be configured using multiple
+different mechanisms. In order of preference, those mechanisms are:
+
+1. [Environment variables](environment.md)
+2. [Standard Git configuration files](configuration.md)
+   1. Repository/local configuration (`.git/config`)
+   2. User/global configuration (`$HOME/.gitconfig` or `%HOME%\.gitconfig`)
+   3. Installation/system configuration (`etc/gitconfig`)
+3. Operating system specific configuration store
```

Comment:
maybe "Operating system specific enterprise configuration store"
to tie it back to the title of the section/document ??

I guess I'm looking for a clarifying term to distinguish [3] from [2.3].

Reply:
How about "Enterprise system administrator defaults"